Source/Core/VideoCommon/BPStructs.cpp

```diff
@@ -265,10 +265,24 @@ static void BPWritten(const BPCmd& bp)
 				ClearScreen(srcRect);
 			}
 
-			return;
+			// Patents indicate the efb has a read bandwidth of 8 pixels per cycle and a write bandwidth of 4 pixels per cycle.
+			// They also indicate that when vertical filtering is enabled (XFB copies), the copy pipeline operates at 4 pixels
+			// per cycle.
+			// If clearing is enabled, the copy operation will be limited to 4 pixels per cycle, but it's possible non-clearing
+			// copies to texture might be able to operate at the full 8 pixels per cycle.
+			// I also suspect the copy pipeline is limited to writing only 64 bits per cycle to memory, which would result in
+			// four 16 bit pixels or two 32 bit pixels per cycle.
+
+			// We will just return the absolute theoretical minimum times (above cycles are GPU cycles, multiply by 3 to get CPU cycles)
+			// Doesn't take into account anything like alignment, just the raw number of pixels.
+			// TODO: Hardware tests to confirm the actual copy speeds of each texture format.
+			if (PE_copy.clear || PE_copy.copy_to_xfb || PE_copy.tp_realFormat() == GX_TF_RGBA8)
```

Comment:
Yes, that would allow a faster RGBX8 copy.

I really want to do some hardware tests to see if the full 8 pixels per cycle efb bandwidth is available.